index.html

```diff
@@ -0,0 +1,180 @@
+            </ul>
+          </div>
+          <div class="navbar-mobile-open">
+            <img src="./images/Icon - Menu.svg" alt="">
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <section class="home">
+      <div class="home-text">
+        <h2>"The IT World Needs You"</h2>
+        <h1>We Build, We Develop</h1>
+        <p class="home-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatem enim obcaecati praesentium veniam nisi suscipit error, modi quisquam eaque fugit asperiores at itaque amet blanditiis mollitia pariatur?</p>
+        <div class="date-time">
+          <h3>2023.07.05(WED) ~ 06(THU)</h3>
+          <p>@ iHub, Senteu Plaza Nairobi</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="main-programs">
+      <h2>Our Programs</h2>
+      <div class="program-cards">
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/react-2.svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>React</h4>
+          </div>
+          <div class="logo-description">
+            <p>React is a free JavaScript library for building user interfaces based on components.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/rails-1.svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Ruby on Rails</h4>
+          </div>
+          <div class="logo-description">
+            <p>Rails is a framework that provides default structures for databases.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/devops-2 (1).svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Devops</h4>
+          </div>
+          <div class="logo-description">
+            <p>DevOps improves and shortens the systems development life cycle.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/kotlin-1.svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Kotlin</h4>
+          </div>
+          <div class="logo-description">
+            <p>Kotlin is a cross-platform high-level programming language with type inference.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/vue-9.svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Vue.js</h4>
+          </div>
+          <div class="logo-description">
+            <p>Vue.js is a JavaScript framework for building user interfaces and single-page applications.</p>
+          </div>
+        </div>
+      </div>
+      <a href="#">Join Our Program</a>
+    </section>
```

Comment:
- [ ] Good job so far :clap: But I think there are a few things with your design you need to work on. Please make sure that when the topic card has hovered with the mouse you have a border around it like on the screenshot below and also your button  `Join our program` has the same color as on the design. :+1: 

- [ ] Please make sure to have the same design, especially the space between your program card as defined on the original design :+1: 

#### Orignal design
![Template2](https://user-images.githubusercontent.com/6615394/210799135-2796a81c-2ff1-4950-9d6b-1f7687384658.jpg)

#### Your design
![progam](https://github.com/NewtonOmbese/Newton-Capstone-Project/assets/6615394/de54185d-a67b-4123-8db2-9e2d8a255545)


- [ ] Please make sure that your program section mobile version design and button are similar to the original. You can see a lot of differences in the screenshots below :point_down: 

#### Original design
![mobk](https://user-images.githubusercontent.com/6615394/210889991-90c15c91-2920-4058-b09a-95e9944af656.jpg)

#### Your design
![mob](https://github.com/NewtonOmbese/Newton-Capstone-Project/assets/6615394/e91714be-a8d7-412a-b454-2c0af9c4c402)

Reply:
Fixed :100: